src/client/components/componentWithSliders/modalsavewindow/savePressetModalBlock.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import ModalSaveWindow from './modalsavewindow';
+import BackgroundForModal from './backgroundformodal';
+
+
+class SavePressetModalBlock extends Component {
+
+    state = {
+        showModalBlock: true
+    }
+    componentDidMount(){
+      const buttonSave = document.getElementById("savePressetes");
+            buttonSave.addEventListener("click", ()=>{
+              this.showHideModalBlock();
+      })
+    }
+
+    showHideModalBlock = () => {  
+      console.log('click')    
+      this.setState({
+        showModalBlock: !this.state.showModalBlock
+      })
+    } 
+       
+
+    render () {
+      const modalBlock = this.state.showModalBlock ? (
+        <React.Fragment>                     
```

Comment:
You can use destructuring this case, like:
`import React, { Component, Fragment } from 'react';`
and use just Fragment wrapper

Reply:
ok